runtime/lib.rs

```diff
@@ -84,7 +84,7 @@ pub static UNSTABLE_GRANULAR_FLAGS: &[UnstableGranularFlag] = &[
   UnstableGranularFlag {
     name: ops::http::UNSTABLE_FEATURE_NAME,
     help_text: "Enable unstable HTTP APIs",
-    show_in_help: false,
+    show_in_help: true,
```

Comment:
This was a mistake to hide it, we still need it for `Deno.createHttpClient()` API that is used heavily in tests